src/components/DpDataTable/DpDataTable.vue

```diff
@@ -170,6 +170,7 @@
 </template>
 
 <script>
+import { de } from '../shared/translations'
 import { CleanHtml } from '../../directives/CleanHtml/CleanHtml'
```

Comment:
alphabetic order

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/429/commits/d4c7d53e26757f641a9ec7bfff88929431dc8a9b